src/dirigera/devices/controller.py

```diff
@@ -5,7 +5,7 @@
 
 
 class ControllerAttributes(Attributes):
-    is_on: bool
+    is_on: bool = None
```

Comment:
please use the same syntax as with other optionals here:
`
Optional[bool] = None
`